src/core_write.cpp

```diff
@@ -223,6 +223,28 @@ void TxToUniv(const CTransaction& tx, const uint256& hashBlock, UniValue& entry,
             const CTxOut& prev_txout = txundo->vprevout[i].out;
             amt_total_in += prev_txout.nValue;
         }
+        if (nullptr != txundo) {
+            switch (verbosity) {
+            case TxVerbosity::DONT_SHOW_TX_DETAILS:
+            case TxVerbosity::TX_DETAILS:
+                break;
+
+            case TxVerbosity::TX_DETAILS_AND_PREV_OUT:
+                const Coin&   prev_coin  = txundo->vprevout[i];
+                const CTxOut& prev_txout = prev_coin.out;
+                UniValue p(UniValue::VOBJ);
+                UniValue o_script_pub_key(UniValue::VOBJ);
+
+                p.pushKV("value", ValueFromAmount(prev_txout.nValue));
+                p.pushKV("height", static_cast<uint64_t>(prev_coin.nHeight));
+                p.pushKV("generated", !static_cast<bool>(prev_coin.fCoinBase));
```

Comment:
Why did you invert this? That is wrong?

Reply:
i was confused as to what "generated" means. Why don't we just call that field "coinbase" ?

Reply:
The coinbase is the *input* to the generation transaction (height + 95 bytes arbitrary). The *outputs* (which are being spent in the input here) are simply generated coins.

Reply:
ah i see. this is fixed now, and i modified the tests to reflect this too.

Reply:
(Please mark as resolved 🙏 )